tutorials/tools/Multispeaker_Simulator.ipynb

```diff
@@ -125,8 +125,8 @@
     "  --input_manifest_filepath LibriSpeech/dev_clean.json \\\n",
     "  --base_alignment_path LibriSpeech_Alignments \\\n",
     "  --output_manifest_filepath ./dev-clean-align.json \\\n",
-    "  --ctm_directory ./ctm_out \\\n",
-    "  --dataset dev-clean"
+    "  --ctm_output_directory ./ctm_out \\\n",
+    "  --libri_dataset_split dev-clean"
```

Comment:
@stevehuang52 Thank you so much that you took care of notebook tutorials too. 
Was the notebook file running without a problem?

Reply:
Haven't run the notebook, but I ran your shell script which has almost the same (with additional args specified) call as this one, so I assume this should work too.

Reply:
I will check the notebook and approve this.

Reply:
Checked that notebook is working fine.